scss/modules/_datatables.scss

```diff
@@ -115,10 +115,15 @@
     background-color: $inverse;
     border-bottom: 1px solid $neutral;
 
+    &.row-color-contrast,
     &:nth-child(even) {
       background-color: rgba($gray-lightest, .5);
     }
 
+    &.row-color-normal {
```

Comment:
Sure, this is so that you can override the `nth-child(even)` rule on line 119 which sets every other row to background gray. The main use case is when cells span multiple rows, like some of the tables in the legal pages. With the `nth-child` rule, you get something like this:

(This only works because github has the same `nth-child` rule for markdown tables)

<table>
<tbody>
<tr>
  <td rowspan="2">I span 2 rows</td>
  <td>one row</td>
</tr>
<tr style="background-color: #eee;">
  <td>one row</td>
</tr>
<tr>
  <td rowspan="3">I span 3 rows</td>
  <td>one row</td>
</tr>
<tr  style="background-color: #eee;">
  <td>one row</td>
</tr>
<tr>
  <td>one row</td>
</tr>
</tbody>
</table>

Markup:
```html
<table>
<tbody>
<tr>
  <td rowspan="2">I span 2 rows</td>
  <td>one row</td>
</tr>
<tr>
  <td>one row</td>
</tr>
<tr>
  <td rowspan="3">I span 3 rows</td>
  <td>one row</td>
</tr>
<tr>
  <td>one row</td>
</tr>
<tr>
  <td>one row</td>
</tr>
</tbody>
</table>
```

You can see how it's still highlighting the even rows, but that's not actually what we want. We really want it to consider the first column cell to be considered a row.

So to apply the new classes, the even rule still applies, unless you override it with `.row-color-normal`. So this markup would get you white-gray alternating at the row group level the way the legal mockups are (sorry, github won't let me demo this one).

Markup
```html
<table>
<tbody>
<tr class="row-color-normal">
  <td rowspan="2">I span 2 rows</td>
  <td>one row</td>
</tr>
<tr class="row-color-normal">
  <td>one row</td>
</tr>
<tr class="row-color-contrast">
  <td rowspan="3">I span 3 rows</td>
  <td>one row</td>
</tr>
<tr class="row-color-contrast">
  <td>one row</td>
</tr>
<tr class="row-color-contrast">
  <td>one row</td>
</tr>
</tbody>
</table>
```

Does that makes sense?